streams/test-utils/src/main/java/org/apache/kafka/streams/test/ConsumerRecordFactory.java

```diff
@@ -246,7 +286,23 @@ public void advanceTimeMs(final long advanceMs) {
     public ConsumerRecord<byte[], byte[]> create(final String topicName,
                                                  final V value,
                                                  final long timestampMs) {
-        return create(topicName, null, value, timestampMs);
+        return create(topicName, null, value, new RecordHeaders(), timestampMs);
+    }
+
+    /**
+     * Create a {@link ConsumerRecord} with {@code null}-key and the given topic name, value, headers, and timestamp.
+     *
```

Comment:
Add missing: `Does not auto advance internally tracked time.`

Reply:
ack.